Resources/views/CRUD/list__select.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends admin.getTemplate('base_list_field') %}
 
 {% block field %}
-    <a class="btn btn-default" href="{{ admin.generateObjectUrl('edit', object) }}">
+    <a class="btn btn-default" href="{{ admin.generateObjectUrl('list', object) }}">
```

Comment:
i think that was the point of the comment. when the route is changed to the list route, the list route does not need the object.

when is this block used? the problem i see with changing this from edit to list is that we now don't link to the edit view of a specific object, but instead to the list of objects which is less convenient. can we find a better solution for the problem? is it maybe more of a documentation issue that we need to explain what to overwrite when removing/renaming the edit route?